src/styles/animations.less

```diff
@@ -0,0 +1,38 @@
+@keyframes simpleSlideUp {
+  0% {
+    margin-top: 0%;
+    opacity: 1;
+  }
+  100% {
+    margin-top: -50%;
+    opacity: 0;
+  }
+}
+@keyframes simpleSlideDown {
+  0% {
+    margin-top: -50%;
+  }
+  100% {
+    margin-top: 0%;
+    opacity: 1;
+  }
+}
+
+@keyframes modalSlideDown {
+  0% {
```

Comment:
Do we need all four of these annimations?  I'm not sure the 'simple' animations are being used.
I see in overlay-panel.less:
```
.catalogs-overlay-window {
  &.ng-enter {
    animation: simpleSlideDown 1s ease forwards;
  }
  &.ng-leave {
    animation: simpleSlideUp 1s ease forwards;
  }
}
```
However, I don't see the class 'catalogs-overlay-window' used anywhere, so we maybe able to get rid of the 'simple' animations.